spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -25,11 +25,11 @@ To create these stream definitions you use the shell or make an HTTP POST reques
 
 == Stream DSL
 
-In the examples above, we connected a source to a sink using the pipe symbol `|`. You can also pass properties to the source and sink configurations. The property names will depend on the individual app implementations, but as an example, the `http` source app exposes a `server.port` setting which allows you to change the data ingestion port from the default value. To create the stream using port 8000, we would use
+In the example above, we connected a source to a sink using the pipe symbol `|`. You can also pass properties to the source and sink configurations. The property names will depend on the individual app implementations, but as an example, the `http` source app exposes a `server.port` setting and it allows you to change the data ingestion port from the default value. To create the stream using port 8000, we would use
 ```
 dataflow:> stream create --definition "http --server.port=8000 | log" --name myhttpstream
 ```
-The shell provides tab completion for application properties and also the shell command `app info` provides some additional documentation.
+The shell provides tab completion for application properties and also the shell command `app info <appType>:<appName>` provides additional documentation for all the supported properties.
```

Comment:
Add a note specifying available app types:

NOTE: Supported appTypes are: source, processor, sink and task.